packages/vscode-extension/tests/specs/api/autoCompletion.spec.ts

```diff
@@ -188,6 +233,35 @@ suite('Auto completion spec', () => {
     });
   });
 
+  test('Does not complete properties for non node variables', async () => {
```

Comment:
The test content can stay the same, but like I mentioned above, rels can have properties too, so I figure this name is more true to what we want
```suggestion
  test('Does not complete properties for non node/relationship variables', async () => {
```